projects/schematics/src/shared/lib-configs/integration-libs/opf-schematics-config.ts

```diff
@@ -125,7 +261,7 @@ function buildOpfConfig(
       ],
       content: `<${OPF_CONFIG}>{
           opf: {
-            baseUrl: "${options.baseUrl || 'PLACEHOLDER_OPF_BASE_URL'}",
+            baseUrl: "${options.commerceCloudApiUrl || options.baseUrl || 'PLACEHOLDER_OPF_BASE_URL'}",
```

Comment:
added a new parameter 'commerceCloudApiUrl' to avoid conflict with 'baseUrl' when installing from top level schematics.
Tested successfully with:
`ng add @spartacus/schematics@latest --base-url https://api.cp96avkh5f-integrati2-d2-public.model-t.cc.commerce.ondemand.com --base-site=electronics-spa --commerceCloudPublicKey=k2N3m3TJPLragwia5ZUvS/qkIPVQoy5qjUkOAB6Db+U= --commerceCloudApiUrl=https://cp96avkh5f-integrati2-d2.opf.commerce.stage.context.cloud.sap/commerce-cloud-adapter-stage/storefront`

Reply:
Probably we don't wanna to have `options.baseUrl` part, because it means that CCv2 url will be provided here. Ideally would be great to unify this config property with the OPF integration lib code.

Would you mind to change it to:
    `commerceCloudApiUrl: "${options.commerceCloudApiUrl || 'PLACEHOLDER_OPF_BASE_URL'}",`?

Then `baseUrl` property should be adjusted in 11 files inside OPF lib. Seems to be not a big effort, however I totally understand if you would like to not extend a scope of this ticket.

Reply:
thx, naming opfBaseUrl instead of 'CommerceCloud..' as it related more to cx-server. It avoid confusion
ng add [@spartacus]/schematics[@latest] --base-url https://api.cp96avkh5f-integrati2-d2-public.model-t.cc.commerce.ondemand.com --base-site=electronics-spa --opf-base-url=k2N3m3TJPLragwia5ZUvS/qkIPVQoy5qjUkOAB6Db+U= --commerceCloudApiUrl=https://cp96avkh5f-integrati2-d2.opf.commerce.stage.context.cloud.sap/commerce-cloud-adapter-stage/storefront